pkg/users/local.js

```diff
@@ -47,6 +47,24 @@ const sortGroups = groups => {
     });
 };
 
+export function get_next_available_id(already_used, is_groups, change) {
+    //  Handle also the case where logindef == null, i.e. the file does not exist.
+    //  While that's unusual, "empty /etc" is a goal, and it shouldn't crash the page.
+    cockpit.file("/etc/login.defs").read()
+            .then(logindef => {
+                const ids = already_used.map(a => is_groups ? a.gid : a.uid);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18353-20230223-103555-bf480ffc-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)